src/components/EventCard/FeaturedEventCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  CardButtonContainer,
+  CardContainer,
+  CardImage,
+  CardTextContainer,
+  CardTitleContainer,
+  Circle,
+  EventClub,
+  EventName,
+  EventRegisterButton,
+  EventTimeVenue,
+  KnowButton,
+} from './styles';
+
+const FeaturedEventCard = () => (

```

Comment:
Try to pass props instead of hardcoded values for ease of integration afterward.